src/main/java/bookmarked/command/AddCommand.java

```diff
@@ -36,9 +42,10 @@ public AddCommand(String newItem, ArrayList<Book> listOfBooks, String[] splitIte
     public void handleCommand() {
         assert newItem != null : "Item should not be null";
         String[] newSplitBook = this.newItem.split("add");
+
         try {
-            processAddCommand(newSplitBook, listOfBooks);
-            assert newSplitBook.length >= 1 : "There should be an argument to the command";
+            processAddCommand(newSplitBook);
+            //assert newSplitBook.length >= 1 : "There should be an argument to the command";
```

Comment:
would it be better to remove the line completely rather than commenting out? since it doesn't explain anything about the code